README.md

```diff
@@ -270,4 +270,4 @@
 
 #### Don't Forgot to share with Your Friends 
 ### The new Update get will soon stay updated
-#### Thank you..!!
+#### Thank you..!! BEST FOR YOU!!
```

Comment:
I don't think this is required!

Reply:
well it looks surely good enough fr the time being